action/Place.action

```diff
@@ -56,6 +56,8 @@ string[] trajectory_descriptions
 # The successful place location, if any
 PlaceLocation place_location
 
+# The amount of time in seconds it took to complete the plan
```

Comment:
This seems ambiguous. Is it the time to plan or the time to execute the plan? Maybe reword to "it took to generate a plan"